### PR TITLE
Add details to get-sposite for the -GroupIdDefined parameter

### PR DESCRIPTION
**Pull request by reedpamsft**
-GroupIdDefined is a parameter that can be added to get-sposite to filter the list of sites returned based on whether they have a group ID or not.

### CODE REVIEW

**Comment by trent-green** [comment]
@get-itips can you review and merge this one as well?

File: sharepoint/sharepoint-ps/sharepoint-online/Get-SPOSite.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ Get-SPOSite [[-Identity] <SpoSitePipeBind>] [-Detailed] [-Limit <String>] [<Comm
 
 ```powershell
 Get-SPOSite [-Detailed] [-Filter <String>] [-IncludePersonalSite <Boolean>] [-Limit <String>]
- [-Template <String>] [<CommonParameters>]
+ [-Template <String>] [--GroupIdDefined] [<CommonParameters>]
```
Comment (get-itips) [inline sharepoint/sharepoint-ps/sharepoint-online/Get-SPOSite.md:31]:
```suggestion
 [-Template <String>] [-GroupIdDefined] [<CommonParameters>]
```

**Review by dariomws — APPROVED** [review]

File: sharepoint/sharepoint-ps/sharepoint-online/Get-SPOSite.md [hunk 2]
```diff
@@ -137,6 +138,15 @@ Get-SPOSite -Limit ALL | ?{$_.IsTeamsChannelConnected -eq $true}
 
 This example uses client-side filtering to return a list of sites connected to a Microsoft Teams Private or Shared channel.
 
+
+### -----------------------EXAMPLE 10-----------------------------
+
+```powershell
+Get-SPOSite -Limit ALL -GroupIdDefined $true
+```
+This example uses server-side filtering to return all sites that have an associated Office 365 Group.
```
Comment (tseward) [inline sharepoint/sharepoint-ps/sharepoint-online/Get-SPOSite.md:147]: Should be "Microsoft 365 Group".

File: sharepoint/sharepoint-ps/sharepoint-online/Get-SPOSite.md [hunk 3]
```diff
@@ -306,6 +316,25 @@ Position: Named
 Default value: None
 ```
 
+### -GroupIdDefined
+
+Filters the list of sites returned to sites with a Group ID (ie: Sites connected to an Office 365 Group) when the value is set to $true.  Filters the list of sites to only sites without a Group ID when the value is $false.
```
Comment (tseward) [inline sharepoint/sharepoint-ps/sharepoint-online/Get-SPOSite.md:321]: Office -> Microsoft 365 Group

**Review by tseward — CHANGES_REQUESTED** [review]

**Review by tseward — APPROVED** [review]

**Comment by yogkumgit** [comment]
@get-itips

**Comment by yogkumgit** [comment]
@get-itips Gentle reminder

**Review by get-itips — APPROVED** [review]

File: sharepoint/sharepoint-ps/sharepoint-online/Get-SPOSite.md [hunk 4]
```diff
@@ -306,6 +316,25 @@ Position: Named
 Default value: None
 ```
 
+### -GroupIdDefined
+
+Filters the list of sites returned to sites with a Group ID (ie: Sites connected to an Microsoft 365 Group) when the value is set to $true.  Filters the list of sites to only sites without a Group ID when the value is $false.
+
+The values are $true, $false, and not defined. By default, the value is not defined which means that the filter does not apply.
```
Comment (tiburd) [inline sharepoint/sharepoint-ps/sharepoint-online/Get-SPOSite.md:323]:
```suggestion
The values are **$true**, **$false**, and **not defined**. By default, the value is **not defined**, which means that the filter does not apply.
```

**Pull request merged**